.travis.yml

```diff
@@ -13,6 +13,7 @@ before_install:
 - echo yes | android update sdk --filter platform-tools --no-ui --force > /dev/null
 - echo yes | android update sdk --filter android-28 --no-ui --force > /dev/null
 - echo yes | android update sdk --filter build-tools-28.0.3 --all --no-ui --force > /dev/null
+- pod repo update
```

Comment:
this should ensure that the pod spec repo is fresh. a likely reason we didnt catch #32 quickly is because the local tavis repo could have used a cached version of 1.28.5 instead of the broken 1.28.6